baybe/utils/scaling.py

```diff
@@ -0,0 +1,22 @@
+"""Scaling utilities."""
+
+from __future__ import annotations
+
+from typing import Protocol
+
+import pandas as pd
+
+
+class ScalerProtocol(Protocol):
+    """Type protocol specifying the interface scalers need to implement.
+
+    The protocol is compatible with sklearn scalers such as
+    :class:`sklearn.preprocessing.MinMaxScaler` or
+    :class:`sklearn.preprocessing.MaxAbsScaler`.
+    """
+
+    def fit(df: pd.DataFrame, /) -> None:
+        """Fit the scaler to a given data set."""
```

Comment:
Would explicitly call it `data frame` instead of `data set`.

Reply:
done